Common/DtaOptions.cpp

```diff
@@ -144,6 +144,10 @@ uint8_t DtaOptions(int argc, char * argv[], DTA_OPTIONS * opts)
 			opts->output_format = sedutilNormal;
 			outputFormat = sedutilNormal;
 		}
+		else if (!strcmp("-c", argv[i])) {
+			baseOptions += 1;
+			g_compat_bsd = 1;
+		}
```

Comment:
I don't like using short option for this. It increases chance of upstream conflict, while this should be relatively low-profile workaround option.

Reply:
Long options usually have double dash in front.

Reply:
Indeed, though `sedutil` refers to these as actions, it allows only one action per command. I can update though if you advise.

Reply:
Absolutely, this parser even don't allow us to combine short options or change their position, i.e., placing after actions.